src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```
@@ -173,7 +183,13 @@ private void ReceiveReplationBatches()
                         catch (Exception e)
                         {
                             if (_log.IsInfoEnabled)
-                                _log.Info("Received unexpected exception while receiving replication batch. This is not supposed to happen.", e);
+                            {
+                                if(e.InnerException is SocketException)
+                                    _log.Info("Failed to read data from incoming connection. The incoming connection will be closed and re-created.", e);
```

This is actually pretty expected, no? Shouldn't we ignore this error entirely? Or report this as just connection closed?